### PR TITLE
Added a coordinate display to the map editor

### PR DESCRIPTION
**Pull request by Mailaender**
See the label widget at the top of the screen.

### CODE REVIEW

**Comment by abcdefg30** [comment]
:+1: / :white_check_mark:

File: OpenRA.Mods.Common/Widgets/Logic/Editor/MapEditorLogic.cs [hunk 1]
```diff
@@ -61,6 +61,13 @@ public MapEditorLogic(Widget widget, World world, WorldRenderer worldRenderer)
 					selectedLabel = selectedZoom.ToString();
 				};
 			}
+
+			var coordinateLabel = widget.GetOrNull<LabelWidget>("COORDINATE_LABEL");
+			if (coordinateLabel != null)
+			{
+				var shape = Game.ModData.Manifest.TileShape;
+				coordinateLabel.GetText = () => worldRenderer.Viewport.ViewToWorld(Viewport.LastMousePos).ToMPos(world.Map).ToCPos(shape).ToString();
```
Comment (penev92) [inline OpenRA.Mods.Common/Widgets/Logic/Editor/MapEditorLogic.cs]: :+1: but why all the conversions here? `ViewToWorld()` gives you a `CPos`.

Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/Editor/MapEditorLogic.cs]: You are right. Those are unnecessary. I copy/pasted old code. Removed.

**Comment by penev92** [comment]
Nice one :+1:

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/3911aeca577bf3cb3ece4c28c49b6897772fd3a5...4ab2848666ce29abc31b2318937caa2c33a17f09)

**Pull request merged**